packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/elements/LineElementProps.java

```diff
@@ -100,7 +100,7 @@ public Builder initialHeight(Integer initialHeight) {
         }
 
         public Builder height(Integer height) {
-            this.height = Objects.requireNonNull(height);
+            this.height = height; // height could be null, in this case, let the table use default height for this line
```

Comment:
No, reset it back to what the specifier want, They are in control.

Reply:
Done